src/Prefix.jl

```diff
@@ -332,28 +364,34 @@ function setup(source::SetupSource{ArchiveSource}, targetdir, verbose; tar_flags
                 libpath = vcat(XZ_jll.LIBPATH_list, libpath)
             end
             if Zstd_jll.is_available()
-                path = vcat(dirname(Zstd_jll.zstd_path), path)
-                libpath = vcat(XZ_jll.LIBPATH_list, libpath)
+                # Zstd_jll became a stdlib in Julia v1.13.0 and the path variable changed name
+                zpath = isdefined(Zstd_jll, :libzstd_path) ? Zstd_jll.libzstd_path : Zstd_jll.zstd_path
```

Comment:
Sigh.  But also, this doesn't look right: the library and the executable are in two different directories.